third-party/sqlalchemy-enum-tables/enumtables/alembic_autogen.py

```diff
@@ -36,12 +30,12 @@ def compare_enums(autogen_context, upgrade_ops, schema_names):
 			metadatas = autogen_context.metadata
 		enums = get_declared_enums(metadatas, schema, default)
 		for table, values in enums.items():
-			fully_qualified_table_name = "%s.%s" % (schema, table) if schema else table
-			if is_table_present(fully_qualified_table_name, autogen_context.connection):
+			if (schema, table) in known_tables:
 				items = {
 					r[0]
 					for r in autogen_context.connection.execute(
-						"SELECT item_id FROM {}".format(fully_qualified_table_name)
+						"SELECT item_id FROM {}".format(
+							"%s.%s" % (schema, table) if schema else table)
```

Comment:
It looks like `get_connection_tables` always has schema table. It uses the default schema if one isn't provided. Is there a case where the `if schema else table` would happen where there isn't a schema?

Reply:
Yes, you're right.

Reply:
Fixed, and used [alembic.autogenerate.render._ident](https://github.com/sqlalchemy/alembic/blob/92b4ed39147aca1d102a2f92bb8dd78580c92c1f/alembic/autogenerate/render.py#L477) to escape.  cc: @phoenixAja